### PR TITLE
Bump GKE containerd versions in the testgrid

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by yujuhong** [comment]
Missed this in the previous PR :(

/assign @Random-Liu

File: testgrid/config.yaml [hunk 1]
```diff
@@ -4085,23 +4085,23 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-cos-k8sstable1-serial
   - name: gci-gke-slow-1.10
```
Comment (krzyzacy) [inline testgrid/config.yaml]: wanna also bump those? maybe add this dashboard to auto-update list as well..

Comment (yujuhong) [reply testgrid/config.yaml]: Done.

File: testgrid/config.yaml [hunk 2]
```diff
@@ -4054,42 +4054,42 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-canary
   - name: gke-cos-containerd-alpha-cluster
     test_group_name: ci-kubernetes-e2e-cos-containerd-gke-alpha-cluster
-  # K8S 1.8 branch on GKE
-  - name: gci-gke-ingress-1.8
+  # K8S 1.9 branch on GKE
```
Comment (krzyzacy) [inline testgrid/config.yaml]: (it won't update comments.... hummm... let's see)

File: experiment/fix_testgrid_config.py [hunk 3]
```diff
@@ -47,6 +47,7 @@
     "sig-release-1.",
     "sig-release-master-upgrade",
     "gke-cos-containerd",
+    "gci-gke",
```
Comment (krzyzacy) [inline experiment/fix_testgrid_config.py]: google-gke?

File: testgrid/config.yaml (hunk 2 above)
Comment (yujuhong) [reply testgrid/config.yaml]: The comments are pretty useless. I updated them to use stableN/beta, etc.

File: experiment/fix_testgrid_config.py (hunk 3 above)
Comment (yujuhong) [reply experiment/fix_testgrid_config.py]: Uh...I misunderstood the naming. Then google-gke should be sufficient for both `gke-cos-containerd` and `gci-gke`. Updated.

**Pull request merged**